SPECS/python-tensorboard/dockerfile

```diff
@@ -9,12 +9,6 @@ FROM mcr.microsoft.com/cbl-mariner/base/core:2.0
 ARG TB_VERSION
 
 WORKDIR /root/
-# RUN tdnf -y install createrepo
-# COPY RPMS /opt/rpms
-# COPY custom.repo /etc/yum.repos.d
-
-# COPY toolchain_rpms /opt/rpms
-# RUN createrepo /opt/rpms
 #bazel version must match with the .bazelversion

```

Comment:
Thanks for the update. Please also separate the source and the cache. From our offline discussion:

I've noticed that if instead of `fetch` I run `sync`, then I was able to run build with the `--nofetch` argument. And I believe with `sync` you won't need to start `build` at all in the Docker container. In other words, I'd try this out:
- In the Dockerfile:
    - Replace `fetch` with `sync`. Also remove the `//tensorboard/pip_package:build_pip_package` target, since `sync` won't like it.
    - Remove the `build` command completely along with the loop.
    - Tar only the `tb_tmp` folder.
- In the spec:
    - Use the original source tarball.
    - Add the tarball with `tb_tmp` as a separate source.
    - Untar `tb_tmp` into the root build folder.
    - Build with `--output_user_root=./tb_tmp` and `--nofetch`.

With these instruction I was able to build the project with `--nofetch` locally from a clean source tarball with a `tb_tmp` folder created by `sync` in a separate copy of the project.